content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -733,13 +655,13 @@ spec:
 {{% /tab %}}
 {{< /tabs >}}
```

Comment:
@sftim 👋 Netlify is failing:

> 5:42:49 PM: Error: Error building site: "/opt/build/repo/content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md:613:6": closing tag for shortcode 'tab' does not match start tag

I'm guessing it's because the opening statements for these lines got lost in the shuffle.